### ISSUE TITLE
'Pie chart' present in 'Security posture' page is not Keyboard accessible. 

### ISSUE THREAD

**Issue by msftedad** [przpiw/Nivo-PieChart-Demo#1]
**Severity** 
Sev2 (High)

**Test Environment** 
OS: Windows 11 22H2
OS Build: 23620.1000

**Repro steps**
1.Open the Above URL: [Pie chart | nivo](https://nivo.rocks/pie/)
2.Pie chart page will appear on the screen
3.Now try to navigate to pie chart and observe the issue whether pie chart is accessible or not.

**Actual Result**
The 'Pie Chart' present in 'Pie' page is not Keyboard accessible.

**Expected Result**
The 'Pie chart' present 'pie' page should be Keyboard accessible.
And Screen reader focus should move on pie chart content and announce the chart data information.

**Attachment**
https://github.com/przpiw/Nivo-PieChart-Demo/assets/93735775/18ead794-f158-4ec2-b1d3-af3657013236
